server/src/db/filesystem_db.rs

```diff
@@ -42,6 +42,25 @@ impl FileSystemDb {
         }
         path
     }
+
+    fn get_directory_content<T: DbEntity>(&self, user_id: &Id<User>) -> DbResult<Vec<T>> {
+        let entity_dir = std::fs::read_dir(self.get_path::<T>(user_id, None));
+        let entity_dir = match entity_dir {
+            Ok(dir) => dir,
+            Err(e) => match e.kind() {
+                std::io::ErrorKind::NotFound => return Ok(Vec::new()),
+                _ => return Err(DbError::WhoopsieDoopsie),
+            },
+        };
```

Comment:
The entity directories must exist for every user. If it does not, our database is corrupted. Remove this error handling and replace it with one `whoopsie`.

Reply:
ping